openedx/core/lib/xblock_builtin/xblock_video/xblock_video/video_xfields.py

```diff
@@ -12,7 +14,7 @@
 
 
 class VideoFields(object):
```

Comment:
For the CapaXBlock PR, we determined that XBlock fields should live in the base class, else they are treated differently than expected.  cf https://github.com/open-craft/edx-platform/pull/145/commits/0c120c58631fd18283761d437a30b51bbc13edc7

Reply:
Also, if any fields need to be exported as separate XML nodes (e.g. `transcripts`), then use [`xml_node=True`](https://github.com/edx/XBlock/blob/7cb80f5dec610cb577c49ca7ee6c6c844fd33c3d/xblock/fields.py#L307-L308).